.github/workflows/tests.yml

```diff
@@ -11,6 +11,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: flake8
+        run: |
+          pip install `egrep -o 'flake8==\S+' web/requirements.txt`  # install our version of flake8
+          flake8 web/ --config web/setup.cfg
+          flake8 docker/pandoc-lambda/function/ --config web/setup.cfg
+
+      - name: black
+        run: |
+          pip install `egrep -o 'black==\S+' web/requirements.txt`  # install our version of black
+          black --check --diff . # Uses pyproject.toml
+
```

Comment:
True? With this setup, links will run both in the `lints` workflow and again in the `tests` workflow?

Reply:
Ah oops, thanks for the catch